wn/web.py

```diff
@@ -208,6 +209,19 @@ def make_synset(ss: wn.Synset, request: Request, basic: bool = False) -> dict:
     return d
 
 
+# Exception handlers
+
+async def http_exception_handler(request: Request, exc: Exception):
+    status_code = exc.status_code if hasattr(exc, 'status_code') else 500
```

Comment:
This is recreating `getattr` behavior:

```suggestion
    status_code = getattr(exc, 'status_code', 500)
```